components/input/IntlTelInput.scss

```diff
@@ -0,0 +1,7 @@
+.intltelinput-input {
+    border-left-color: transparent !important;
```

Comment:
`!important` because dropdown class styles are applied after these styles. A temporary fix to avoid double border between elements.